presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.
+* Fix a failure when view columns were fully qualified with catalog and/or schema.
```

Comment:
Does `a failure` mean query failure here? If you look at past release notes we commonly use the phrase `Fix query failure when/for`